test/test_utils/__init__.py

```diff
@@ -139,9 +142,24 @@ def get_ami_id_ssm(region_name, parameter_path):
 ECR_SCAN_HELPER_BUCKET = f"""ecr-scan-helper-{boto3.client("sts", region_name=DEFAULT_REGION).get_caller_identity().get("Account")}"""
 ECR_SCAN_FAILURE_ROUTINE_LAMBDA = "ecr-scan-failure-routine-lambda"
 
+## Note that the region for the repo used for conducting ecr enhanced scans should be different from other
+## repos since ecr enhanced scanning is activated in all the repos of a region and does not allow one to 
+## conduct basic scanning on some repos whereas enhanced scanning on others within the same region.
 ECR_ENHANCED_SCANNING_REPO_NAME = "ecr-enhanced-scanning-dlc-repo"
 ECR_ENHANCED_REPO_REGION = "us-west-1"
 
+class NightlyFeatureLabel(Enum):
+    AWS_FRAMEWORK_INSTALLED = "aws_framework_installed"
+    AWS_SMDEBUG_INSTALLED = "aws_smdebug_installed"
+    AWS_SMDDP_INSTALLED = "aws_smddp_installed"
+    AWS_SMMP_INSTALLED = "aws_smmp_installed"
+    PYTORCH_INSTALLED = "pytorch_installed"
+    AWS_S3_PLUGIN_INSTALLED = "aws_s3_plugin_installed"
+    TORCHAUDIO_INSTALLED = "torchaudio_installed"
+    TORCHVISION_INSTALLED = "torchvision_installed"
+    TORCHDATA_INSTALLED = "torchdata_installed"
```

Comment:
While merging from the master, this portion of code was getting conflicted with the lines 148 and 149 i.e.
```
ECR_ENHANCED_SCANNING_REPO_NAME = "ecr-enhanced-scanning-dlc-repo"
ECR_ENHANCED_REPO_REGION = "us-west-1"
```
To resolve the merge commit, I kept both the changes. 

Looks like git picked the later change as the one that I added. Not sure why did that happen

Reply:
If I open the file and look into the git blame, I see that the lines are properly attributed to the correct developers. Probably its some issue with the way git diff is working on the UI?

Reply:
Got it. I think that it treats it as "code that was committed between the last review and now", because it came from a commit that you made to resolve the merge conflict. If it was a merge commit that was automatically done by git, I guess it would have ignored it.

Now that I look at the total diff between your branch and master, rather than between your last revision and your current revision, that block of code no longer looks like you were the one that made the change.

Thanks!

Reply:
Got it, thanks!!